x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/header/header_content.tsx

```diff
@@ -31,10 +32,11 @@ const HeaderContentComponent: React.FC<HeaderContentProps> = ({
   isLoading,
   isResizing,
   onClick,
+  showSortingCapability,
   sort,
 }) => (
   <EventsHeading data-test-subj={`header-${header.id}`} isLoading={isLoading}>
-    {header.aggregatable ? (
+    {header.aggregatable && showSortingCapability ? (
```

Comment:
It appears that the intent (and effect) of this change disables sorting when the user clicks on a header in `EQL` mode, which makes sense.

Consider (also) disabling the `Sort fields` button in the screenshot below if that also makes sense in `EQL` mode:

<img width="1266" alt="sort-fields-button" src="https://user-images.githubusercontent.com/4459398/107459779-20476580-6b14-11eb-9ba7-8b4262eb4ee7.png">